tests/test_main.py

```diff
@@ -598,6 +598,35 @@ class ModelB(BaseModel):
     assert dict(m) == {'c': 5, 'd': 2, 'e': {'a': 0, 'b': 1}}
 
 
+def test_dict_skip_defaults_populated_by_alias():
+    class MyModel(BaseModel):
+        class Config:
```

Comment:
for clarity I don't think you need `allow_population_by_alias` here

Reply:
also generally it's conventional to put `Config` at the end of the class definition, same as `Meta` in django etc.